src/csharp/Extensions/Microsoft.Spark.Extensions.Delta.E2ETest/DeltaTableTests.cs

```diff
@@ -70,7 +69,7 @@ public void TestTutorialScenario()
                 // |106|
                 // |108|
                 // +---+
-                ValidateDataFrame(
+                ValidateRangeDataFrame(
```

Comment:
Should be enough space to make this one line ?

Reply:
Fixed.